tox.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+skipsdist=True
+usedevelop=True
+
+envlist=dj{18,111}
+
+[testenv]
+install_command=pip install -e ".[testing]" -U {opts} {packages}
+commands=
+     coverage erase
+     coverage run manage.py test {posargs}
```

Comment:
Minor suggestion -- how would you feel about adding another command here, `coverage report`? This would dump to the console the coverage output, which would be helpful both locally and on Travis.

If we don't do this, it is possible to run this command outside of tox, but it means that the user has to have coverage installed in that context. If included here, you can just checkout this repo and run tox and never need to set up a local virtualenv at all.